lib/devices/ios/webkit-remote-debugger.js

```diff
@@ -10,10 +10,11 @@ var _ = require('underscore')
 // CONFIG
 // ====================================
 
-var WebKitRemoteDebugger = function (onDisconnect) {
+var WebKitRemoteDebugger = function (onDisconnect, webkitDebugProxyPort) {
   this.dataMethods = {};
   this.host = 'localhost';
-  this.port = process.env.REMOTE_DEBUGGER_PORT || 27753;
+  //this.port = process.env.REMOTE_DEBUGGER_PORT || 27753;
```

Comment:
If this is no longer needed, it should be deleted, not commented out. 

I am also wary of summarily removing functionality. There needs to be a way to allow people, for now, to use the former functionality.

```js
if (process.env.REMOTE_DEBUGGER_PORT) {
  this.logger.debug('Found remote debugger port through REMOTE_DEBUGGER_PORT environment variable. This is deprecated, use `--webkit-debug-proxy-port` server argument');
  webkitDebugProxyPort = webkitDebugProxyPort || process.env.REMOTE_DEBUGGER_PORT;
}
```